docs/getting-started/introduction.md

```diff
@@ -0,0 +1,78 @@
+
+EventStoreDB also includes additional features that make it simple to develop event-native applications that enable real-time updates and minimize system dependencies while maintaining data consistency.
+
+Finally, EventStoreDB can reconstruct the current state of any object from its historical events, providing the best of both current and historical representations to answer the "what" and "why" within a single system.
+
+::: info Current State and Historical Events Representations
+
+In a system, any object can represented in one of two distinct ways: by its current state, or by the historical events leading up to that state:
+
+- **Current state** is a snapshot of the object right now, like a digital wallet's balance. 
+- **Historical events** are the actions that led to this state, like a wallet's deposits and withdrawals.
+
+Both perspectives are important: the current balance can answer the "what", while historical transactions answer the "why".
+
+To learn more about this topic, [click here](/getting-started/evaluate/state-vs-event-based-data-model.html).
+:::
+
+## Who Uses EventStoreDB?
+
+Data engineers can provide context-rich events from EventStoreDB to data pipelines to analyze historical and behavioral trends, uncovering patterns that traditional databases often miss. These patterns can, for example, reveal why a customer churned or highlight behaviors that lead to high-value contracts.
+
+::: note
+To learn more about EventStoreDB's role in a data pipeline, [click here](/getting-started/evaluate/role-of-eventstoredb-in-a-data-pipeline.html).
+:::
+
+Application developers can leverage EventStoreDB’s granular events to build real-time, distributed enterprise applications and break down tightly coupled systems. Events are typically simpler, self-contained, and independent. Unlike the tables and rows in traditional databases that are often interdependent with various functions.
+
+## Where to Use EventStoreDB?
+
+EventStoreDB is valuable for critical enterprise applications and data pipelines with behavior-rich data, where interdependent business objects and processes interact in diverse ways over time. 
+
+Industries such as finance, healthcare, supply chain, and manufacturing benefit from EventStoreDB, as it supports flexible, scalable solutions with a complete audit trail. This enables businesses to maintain clear, comprehensive system oversight, which is essential for meeting compliance requirements and adapting to changing needs.
+
+::: note
+To learn more where EventStoreDB fits in your business, [click here](/getting-started/evaluate/where-eventstoredb-fits-in-your-business.html).
+:::
+
+## Next Steps
+
+- [Key Benefits of EventStoreDB](/getting-started/evaluate/key-benefits-of-eventstoredb.html): Learn what EventStoreDB can do for you. 
```

Comment:
May want to remove this link, if the plan is still to remove the "key benefits" section